.github/workflows/build-master.yaml

```diff
@@ -0,0 +1,63 @@
+name: build-master
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - '.mvn/**'
+      - 'mvnw'
+      - 'mvnw.cmd'
+      - 'README.md'
+      - '.gitignore'
+
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '.mvn/**'
+      - 'mvnw'
+      - 'mvnw.cmd'
+      - 'README.md'
+      - '.gitignore'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'graalvm'
+          java-package: 'jdk'
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Start Database
+        run: ./setup-database.bash
+
+      - name: Build project
+        run: mvn clean install
+
+      - name: Show test result
+        run: mvn surefire-report:report-only
+             cat target/site/surefire-report.html
+
+      - name: Publish to GitHub Packages
+        run: |
+          cd microservices
+          echo "//maven.pkg.github.com/username/repository/:registryToken=$GITHUB_TOKEN" > .npmrc
+          mvn deploy -Dmaven.deploy.skip=true -Dmaven.repo.local=$HOME/.m2/repository
```

Comment:
Ten maven.deploy.skip=true jest potrzebny tutaj?

Reply:
This is set to true because I think that the "master" branch is a development branch, so we will not build it "in theory" (unless something changes, then just change the flag to false to push the Jar to the github package). I anticipate that we will have a "pre-prod" and "prod" branch, but this still needs to be agreed upon at a meeting.